requirements-dev.txt

```diff
@@ -0,0 +1,9 @@
+requests_oauthlib
+oauthlib
+google_auth_oauthlib
+boto3 >= 1.5.12, < 2
+chalice >= 1.1.0, < 2
+pyjwt >= 1.5.3, < 2
+cryptography >= 2.3, < 3
+furl >= 1.2.1
+pyyaml
```

Comment:
Please use `-r requirements.txt` to transclude it here instead of duplicating the list.

Reply:
will fix